subsys/net/lib/openthread/platform/platform.c

```diff
@@ -21,8 +21,8 @@ void otSysInit(int argc, char *argv[])
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
-	platformAlarmInit();
 	platformRadioInit();
+	platformAlarmInit();
```

Comment:
Why this order is changed? Don't we use timers in the driver?

Reply:
I don't think we use `ot*` timing functions in the radio driver implementation. @lmaciejonczyk knows the details about the need of this change for aligning platform and radio timing.